eventManager.js

```diff
@@ -0,0 +1,39 @@
+// Decides what event should be emitted
+// old_status: the statusType of the asset already in the database, is null if the asset has just been added to the database
+// new_asset_status: contains the asset status we just added to the database
+// eventNames: enum containing all the possible events
+// statusTypes: enum containing all the possible types of asset["statusTypes"]
+// returns: event we want to raise
+module.exports = function eventManager(old_status, new_asset_status, eventNames, statusTypes){

```

Comment:
Why `eventNames` and `statusTypes` are passed in as arguments here? Usually you would pass something as argument if it can change between function calls which is not the case for `eventNames` or `statusTypes`. Since they stay unchanged it would be easier to import them in this module using `require` which would also simplify the `eventManager` function signature:
```suggestion
module.exports = function eventManager(old_status, new_asset_status){
```